src/web/assets/cp/src/js/TableElementIndexView.js

```diff
@@ -19,9 +21,20 @@ Craft.TableElementIndexView = Craft.BaseElementIndexView.extend({
     },
 
     afterInit: function() {
+        // Set table caption
+        this.$tableCaption = this.$table.find('caption');
+
+        this.$statusMessage = this.$table.parent().find('[data-status-message]');
+
         // Set the sort header
         this.initTableHeaders();
 
+        // Add callback for after elements are updated
+        this.elementIndex.settings.onUpdateElements = () => {
```

Comment:
Feel iffy about setting this directly as there might already be something assigned. Is there a better way you can recommend @brandonkelly ?

Reply:
Fixed by just registering an `updateElements` event:

https://github.com/craftcms/cms/blob/0d32e98121cb819d5b1eb1d52955d66c2cf55bb5/src/web/assets/cp/src/js/TableElementIndexView.js#L33